src/examples/aws-cognito/src/backend/index.ts

```diff
@@ -0,0 +1,13 @@
+import 'reflect-metadata';
```

Comment:
```suggestion
```

We shouldn't need `reflect-metadata` anywhere except in the mikro-orm package now.